packages/build-utils/test/unit.get-workspaces.test.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+import { getWorkspaces, Workspace } from '../src/workspaces/get-workspaces';
+import { FixtureFilesystem } from './utils/fixture-filesystem';
+
+describe.each<[Workspace[], string]>([
```

Comment:
Nit pick: I'm not sure the conventions here but I generally think of the input on the left and the output on the right (so this seems backwards)